demos/benchmark/lv_demo_benchmark.h

```diff
@@ -96,6 +96,35 @@ void lv_demo_benchmark_summary_display(const lv_demo_benchmark_summary_t * summa
 /**********************
  *      MACROS
  **********************/
+#if LV_DEMO_BENCHMARK_ALIGNED_FONTS
+#if LV_FONT_BENCHMARK_MONTSERRAT_12_ALIGNED
+LV_FONT_DECLARE(lv_font_benchmark_montserrat_12_aligned)
+#endif
+
+#if LV_FONT_BENCHMARK_MONTSERRAT_14_ALIGNED
+LV_FONT_DECLARE(lv_font_benchmark_montserrat_14_aligned)
+#endif
+
+#if LV_FONT_BENCHMARK_MONTSERRAT_16_ALIGNED
+LV_FONT_DECLARE(lv_font_benchmark_montserrat_16_aligned)
+#endif
+
+#if LV_FONT_BENCHMARK_MONTSERRAT_18_ALIGNED
+LV_FONT_DECLARE(lv_font_benchmark_montserrat_18_aligned)
+#endif
+
+#if LV_FONT_BENCHMARK_MONTSERRAT_20_ALIGNED
+LV_FONT_DECLARE(lv_font_benchmark_montserrat_20_aligned)
+#endif
+
+#if LV_FONT_BENCHMARK_MONTSERRAT_24_ALIGNED
+LV_FONT_DECLARE(lv_font_benchmark_montserrat_24_aligned)
+#endif
+
+#if LV_FONT_BENCHMARK_MONTSERRAT_26_ALIGNED
+LV_FONT_DECLARE(lv_font_benchmark_montserrat_26_aligned)
+#endif
+#endif /*LV_DEMO_BENCHMARK_ALIGNED_FONTS*/
```

Comment:
It can go to the C file

Reply:
I've just checked, if we want it to also be used in the "Widgets demo" test, it must be in the header